rel/files/riak_cs.schema

```diff
@@ -189,7 +195,7 @@
 %% @doc How often to flush the access log; integer factor of
 %% access_archive_period (1 == once per period; 2 == twice per period,
 %% etc.)
-{mapping, "access.stats.log_flush_factor", "riak_cs.access_log_flush_factor", [
+{mapping, "access.stats.log.flush_factor", "riak_cs.access_log_flush_factor", [
```

Comment:
There are two kinds of statistics in riak cs, access and storage. If they are grouped, `stats.access.*` is better name.
If not,  `access_stats` is nice for prefix.